### ISSUE TITLE
Make errors building from source in Debian Testing(Buster)

### ISSUE THREAD

**Issue by Daniel-M** [stevengj/meep#94]
Hello everyone,

I'm trying to build meep from sources from the master branch, (which shows the "build passed" flag at the moment). My machine `uname -a` shows,
```sh
Linux 4.12.0-1-amd64 #1 SMP Debian 4.12.6-1 (2017-08-12) x86_64 GNU/Linux
```
Installed the following prerequisites recommended from the Meep readthedocs page,

```sh
apt install libctl5 libctl-dev guile-2.0-dev guile-2.0 mpi-default-bin mpi-default-dev swig libhdf5-dev libhdf5-cpp-100 libhdf5-100 libhdf5-mpi-dev libhdf5-openmpi-dev h5utils hdf5-tools
```

Once the `./configure --with-mpi` ends y do `make -j4` and the following errors appear,

```sh
bend-flux-ll.cpp:18:10: fatal error: ctl-math.h: No existe el fichero o el directorio(spanish for "file not found")
 #include "ctl-math.h"
```

```sh
meepgeom.cpp:44:26: error: invalid conversion from ‘void*’ to ‘SCM {aka scm_unused_struct*}’ [-fpermissive]
 material_type vacuum = { (void *)&vacuum_material_data };
                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
meepgeom.cpp: In function ‘material_type meep_geom::make_dielectric(double)’:
meepgeom.cpp:61:24: error: invalid conversion from ‘void*’ to ‘SCM {aka scm_unused_struct*}’ [-fpermissive]
   material_type mt = { (void *)md };
                        ^~~~~~~~~~
meepgeom.cpp: In function ‘void meep_geom::epsilon_file_material(meep_geom::material_data*, vector3)’:
meepgeom.cpp:352:25: error: invalid conversion from ‘void*’ to ‘SCM {aka scm_unused_struct*}’ [-fpermissive]
   default_material.data=(void *)md;
                         ^~~~~~~~~~
```
I tried to find `ctl-math.h` but I couldn't find it (`find / -name "ctl-math.h"`)  at standard places like `/usr/include` and even in the whole system and nothing was found.
Since the officially packaged meep for Debian doesn't work either, I find the use or compilation of meep virtually impossible, since the issue #57 is present on my system too.

**Comment by Daniel-M** [comment]
**UPDATE** Installed libctl from the tar file downloadable at http://ab-initio.mit.edu/wiki/index.php/Libctl and the missing header

**Comment by Daniel-M** [comment]
cloned https://github.com/stevengj/libctl and installed libctl from there. This time the meep package compiled without problems with guile v2.0.11 manually installed. Though the compilation is resolved, can't use meep given that the issue #57 is still unresolved even with guile v2.0.11

**Comment by JohnWeiner** [comment]
Make sure you don’t have two instances of guile installed and that meep is invoking 2.0.11 and not a later version.
